src/card/Card.js

```diff
@@ -19,9 +20,15 @@ class Card extends React.Component {
 
     render() {
         var { className, shadowLevel, ...otherProps } = this.props;
+
+        var noShadow = shadowLevel <0;
```

Comment:
using `typeof shadowLevel !== 'undefined'` would be better in this case, so if the property is not provided, we don't put any shadow on it.